src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -1769,6 +1769,14 @@ compiler.note.proc.messager=\
 compiler.note.multiple.elements=\
     Multiple elements named ''{1}'' in modules ''{2}'' were found by javax.lang.model.util.Elements.{0}.
 
+compiler.note.implicit.annotation.processing=\
+    Annotation processing is enabled because one or more processors were found\n\
+    on the class path. A future release of javac may disable annotation processing\n\
+    unless at least one processor is specified by name (-processor), or a search\n\
+    path is specified (--processor-path, --processor-module-path), or annotation\n\
+    processing is enabled explicitly (-proc:only, -proc:full).\n\
+    Use -Xlint:-options to suppress this message.
```

Comment:
You ought to be able to suppress the message by addressing the underlying case, not just by disabling the check.
For example, I would expect `-proc:none` to disable the message as well.

Reply:
That observation is true and I'll add -proc:none to the test case.

I'll add a sentence that -proc:none will disable processing.